CHANGELOG.md

```diff
@@ -192,6 +230,20 @@
     Calendar.render_to_response(deps_strategy="fragment")
     ```
 
+- The `render_dependencies` kwarg in `Component.render()` and `Component.render_to_response()` is now deprecated. Use `deps_strategy="raw"` instead. The `render_dependencies` kwarg will be removed in v1.
+
+    Before:
+
+    ```py
+    Calendar.render_to_response(render_dependencies=False)
+    ```
+
+    After:
+
+    ```py
+    Calendar.render_to_response(deps_strategy="raw")
+    ```
```

Comment:
I'm not super fond of saying "I want to render this with a _raw_ dependency strategy". I agree that "none" isn't great either, because it sounds like you have no strategy. Maybe "ignore" could work? "The strategy is to ignore the dependencies" makes sense I think!

Reply:
Yeah, I like "ignore"!